packages/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -367,9 +376,9 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
         this.captionStatus = TURN_ON_CAPTION_STATUS.ENABLED;
         this.announce();
       })
-      .catch(() => {
+      .catch((e) => {
```

Comment:
Why is 'e' added without using it?

Reply:
Arun had a console log for this `e` that I removed. Removed the `e` also now